### DISCUSSION TITLE
2023 On-boarding process revisions

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/.github#10] [General]
Hey @UXBrendan, in the spirit of moving design/UX closer to dev teams over time, **what do you and the other UX/design team leads think about experimenting here in this `.github` repo as we revise the on-boarding documentation and process for 2023?** 

I'm wondering whether:

1. [Projects listed here](https://github.com/orcasound/.github/projects?query=is%3Aopen)(and maybe managed here by PMs?) could start to replace Trello boards for some teams...
2. An [organization-wide, repo-spanning Wiki](https://github.com/orcasound/.github/wiki) could replace parts of Orcasound's onboarding GoogleDoc/PDF content, among other over-arching documentation and resources. You could make separate wiki page(s) for UX on-boarding after we organize highest level of the wiki.
3. directories here or in new, separate repos could hold progressively more content used by UX/design team. For example, I could imagine `orca-ux` directory or new repo with anonymized user research (e.g. survey results), personas, survey templates and an `orca-design` directory or new repo with assets like logos, favicons, graphics, photos.
4. Maybe Issues and Actions could be useful for other organization-wide processes?
5. I've signed up for Orcasound to beta-test [two new Github features -- Roadmaps and Tasklists](https://github.blog/2022-11-15-the-journey-of-your-work-has-never-been-clearer/)...

**Comment by scottveirs** [comment]
A good question from Paul at today's standup -- where should non-acoustic data backups and dumps live? He was thinking of a backup of the `orcagsoc` Heroku instance database (before deleting the instance to save $$)...

Maybe such things and any private info, spreadsheets could live in a shared Orcasoundapp admin Gdrive folder?

**Comment by UXBrendan** [comment]
@scottveirs 

In response to your post above:

**1. [Projects listed here](https://github.com/orcasound/.github/projects?query=is%3Aopen)(and maybe managed here by PMs?) could start to replace Trello boards for some teams...**
<p>I've created projects geared toward product and UX work in the Product repo- https://github.com/orcasound/product/projects?query=is%3Aopen. Projects have replaced Trello for the UX Team.

**2. An [organization-wide, repo-spanning Wiki](https://github.com/orcasound/.github/wiki) could replace parts of Orcasound's onboarding GoogleDoc/PDF content, among other over-arching documentation and resources. You could make separate wiki page(s) for UX on-boarding after we organize highest level of the wiki.**
<p>I think after new volunteers read the GitHub landing page, they should be directed to onboard via their domain of interest- Machine Learning, Dev, UX, etc. I've created the Product repo for Product and UX onboarding and project management purposes. A similar infrastructure could be set up for other domains like Machine Learning and Dev.

**3. directories here or in new, separate repos could hold progressively more content used by UX/design team. For example, I could imagine orca-ux directory or new repo with anonymized user research (e.g. survey results), personas, survey templates and an orca-design directory or new repo with assets like logos, favicons, graphics, photos.**
<p>I'm not sure how directories work in GitHub- GDrive seems to be a great solution for the UX Team, even though it costs about $20/yr for 100gb of storage. We do use a lot of Google Apps like Docs, Sheets, and especially Forms. Keeping our files in GDrive seems to be the best solution, even though allowing admin access to the orcasound.ux Google account is problematic. Others can easily access the GDrive, but I am concerned with security. I've recently set the global permission level to "Commenter", and volunteers who want edit access to the GDrive will have to be given that from those who have edit access already.

**4. Maybe Issues and Actions could be useful for other organization-wide processes?**
<p>Yes, I've started writing issues and assigning to UX Team members, but they have been hesitant to use GitHub because they view it as a dev platform and think it would be hard to learn. Adoption is actually easy, and I'm planning on providing training to UX Team members as I use GitHub as our project management platform. It's been challenging to get UX Team members to sign up for a GitHub account so I can assign them issues.

**5. I've signed up for Orcasound to beta-test [two new Github features -- Roadmaps and Tasklists](https://github.blog/2022-11-15-the-journey-of-your-work-has-never-been-clearer/)...**
<p>I've been using some of these beta features. Excited to see how GitHub continues to scale their offerings!
